src/code/utilities/guide-protocol.js

```diff
@@ -40,14 +40,24 @@ GuideProtocol.Event = function(username, session, sequence, actor, action, targe
 
 GuideProtocol.Event.Channel = 'Event';
 
+GuideProtocol.Event.prototype.isMatch = function(actor, action, target) {
```

Comment:
Nor me. I just pulled it in as-is from https://github.com/IntelliMedia/guide-server/blame/master/shared/guide-protocol.js